src/ResourceManager/Profile/Commands.Profile/Errors/AzureErrorRecord.cs

```diff
@@ -0,0 +1,42 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Profile.Errors
+{
+    public class AzureErrorRecord
+    {
+        public AzureErrorRecord(ErrorRecord record)
+        {
+            InvocationInfo = record.InvocationInfo;
+            ScriptStackTrace = record.ScriptStackTrace;
+            ErrorCategory = record.CategoryInfo;
+            ErrorDetails = record.ErrorDetails;
+        }
+
+        public ErrorDetails ErrorDetails { get; set; }
+
+        public ErrorCategoryInfo ErrorCategory { get; set; }
```

Comment:
@markcowl nit: can we put an empty line between `ErrorCategory` and `InvocationInfo`?